tools/bumper.sh

```diff
@@ -0,0 +1,96 @@
+#!/bin/bash
+set -euo pipefail
+
+# -------------------------------
+# Wazuh Indexer Version Bump Process
+# -------------------------------
+# This script updates:
+# - VERSION.json: sets new "version" and "stage"
+#
+# Usage: new-version.sh <version> <stage>
+# Example: new-version.sh 1.2.3 alpha
+# -------------------------------
+
+function usage() {
+    echo "Usage: $0 <version> <stage>"
+    echo "  version:        The new version to set in VERSION.json"
+    echo "  stage:          The new stage to set in VERSION.json"
+    exit 1
+}
+
+# Ensure that exactly two arguments are passed
+if [ "$#" -ne 2 ]; then
+    usage
+fi
+
+# Check for required dependencies
+if ! command -v jq &>/dev/null; then
+    echo "Error: 'jq' is not installed. Please install it and re-run the script." >&2
+    exit 1
+fi
+
+VERSION=$1
+STAGE=$2
+VERSION_FILE="VERSION.json"
+BACKUP_DIR="/tmp/wazuh-backups"
```

Comment:
`TMP_DIR` is more convenient for this case. As `jq` does not modify files in place, we need to read, modify and write them elsewhere, to later on overwrite the original file with the modified one.

I would avoid creating a specific folder for this, and write to the `/tmp` directly. In fact, I'd create the modified at the same level of the original one, swap them and clean. The less we need to manage, the better.

In that case, this variable could be removed.